docs/book/src/reference/commands/alpha_update.md

```diff
@@ -81,8 +87,46 @@ Once updated, you can use `kubebuilder alpha update` for future upgrades.
 | `--from-version`    | **Required for projects initialized with versions earlier than v4.6.0.** Kubebuilder version your project was created with. If unset, uses the `PROJECT` file. |
 | `--to-version`      | Version to upgrade to. Defaults to the latest version.                                                                                                         |
 | `--from-branch`     | Git branch that contains your current project code. Defaults to `main`.                                                                                        |
+| `--force`           | Force the update even if conflicts occur. Conflicted files will include conflict markers, and a commit will be created automatically. Ideal for automation (e.g., cronjobs, CI).                                                                       |
 | `-h, --help`        | Show help for this command.                                                                                                                                    |
 
+<h1> Using the `--force` flag </h1>
```

Comment:
It will not render properly 
maybe ## Using the `--force` flag 
Here ^ 

and bellow add one more for ## Handling Merge Conflicts
to be like ### Handling Merge Conflicts